app/buyers/views/buyers.py

```diff
@@ -0,0 +1,279 @@
+from flask import abort, render_template, request, redirect, url_for, flash
+from flask_login import current_user
+
+from app import data_api_client
+from .. import buyers, content_loader
+from ...helpers.buyers_helpers import count_suppliers_on_lot, get_framework_and_lot, is_brief_associated_with_user, \
+    count_unanswered_questions, brief_can_be_edited, add_unanswered_counts_to_briefs
+from ...helpers.search_helpers import get_template_data
+
+from dmapiclient import HTTPError
+
+
+@buyers.route('/buyers')
+def buyer_dashboard():
+    template_data = get_template_data(buyers, {})
+    user_briefs = data_api_client.find_briefs(current_user.id).get('briefs', [])
+    draft_briefs = add_unanswered_counts_to_briefs([brief for brief in user_briefs if brief['status'] == 'draft'])
+    live_briefs = [brief for brief in user_briefs if brief['status'] == 'live']
+
+    return render_template(
+        'buyers/dashboard.html',
+        draft_briefs=draft_briefs,
+        live_briefs=live_briefs,
+        **template_data
+    )
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>', methods=['GET'])
+def info_page_for_starting_a_brief(framework_slug, lot_slug):
+
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client)
+
+    if framework['status'] != 'live':
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    return render_template(
+        "buyers/start_brief_info.html",
+        framework=framework,
+        lot=lot,
+        supplier_count=count_suppliers_on_lot(framework, lot),
+        **dict(buyers.config['BASE_TEMPLATE_DATA'])
+    ), 200
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/create', methods=['GET'])
+def start_new_brief(framework_slug, lot_slug):
+    """Page to kick off creation of a new brief."""
+
+    framework = data_api_client.get_framework(framework_slug)['frameworks']
+    if framework['status'] != 'live':
+        abort(404)
+
+    try:
```

Comment:
This is repeated quite a bit, how about a `framework, lot = get_framework_and_lot(...)` helper function that takes care of this and related lot checks with 404 aborts?